lib/mbta.js

```diff
@@ -1,9 +1,34 @@
 'use strict';
 
-// Code here.
+// Dictionary of line names with arrays representing each subway line.
+const mbtaSystem = {
+  'Red':
+    ["Alewife", "Davis", "Porter", "Harvard", "Central", "Kendall", "Park Street", "South Station"],
+  'Green':
+    ["Kenmore", "Hynes", "Copley", "Arlington", "Boylston", "Park Street", "Government Center"],
+  'Orange':
+    ["Forest Hills", "Back Bay", "Chinatown", "Downtown Crossing", "State", "Park Street", "Haymarket", "North Station"]
+}
 
-const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
+// Returns the stop number of a station given the line name and the station name.
+const stopNum = function(line,station) {
+  return mbtaSystem[line].indexOf(station);
+}
 
+// Finds the stops between two stations on the same line.
+const stopsNoXfr = function(line,stnA,stnB) {
+  return Math.abs(stopNum(line,stnA) - stopNum(line,stnB));
+}
+
+// If start and end lines are the same, calls stopsNoXfr.
+// If start and end lines are different sums the distance between each
+// stop and Park Street on that line.
+const stopsBetweenStations = function(startLine, startStation, endLine, endStation) {
+  if (startLine === endLine) {
+    return stopsNoXfr(startLine,startStation,endStation);
+  } else {
+    return stopsNoXfr(startLine,startStation,"Park Street") + stopsNoXfr(endLine,"Park Street",endStation);
```

Comment:
Is there a better way to do this then passing in the string "Park Street"?  Think on this.